### ISSUE TITLE
[Bug]: float value is interpreted as a string with length requirement

### ISSUE THREAD

**Issue by zargot** [Big-Life-Lab/PHES-ODM-Validation#225]
### Bug Description

error:
`measures(value, 3): Value "9.5228000000000004E-5" (of length 21) exceeds the max length of "15" [greater_than_max_length]`

### The operating system on which the bug occured

linux

### Python Version

3.8

**Comment by yulric** [comment]
Lets ask the rest of the team @mathew-thomson. It makes sense to make it a float I think?

**Comment by zargot** [comment]
then we need to wait for the ODM to be updated, and then generate new schemas. I'm deferring this to v0.7.0.

**Comment by zargot** [comment]
@mathew-thomson, has the ODM been updated with this change (increasing the length of measures.value to >=22) yet?

**Comment by mathew-thomson** [comment]
@zargot yes, that should now be upped to 30 or 50. I think you can consider it resolved.

**Comment by zargot** [comment]
the latest news is that it will be fixed in the next (patch) version of the ODM, so I'm waiting for that one 👍

**Comment by zargot** [comment]
delaying this to the next version, as we want to release now, and this won't be ready before 2-3 weeks from now.
